tests/e2e/utils/auto-rds-s3-cleanup.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+from time import sleep
+import boto3
+
+from utils import get_rds_client, get_s3_client, get_secrets_manager_client
+
+
+def main():
+    metadata_dict = {}
+    with open("auto-rds-s3-setup-metadata", "r") as setup_metadata:
+        for line in setup_metadata.readlines():
+            metadata = line.split("=")
+            metadata_dict[metadata[0]] = metadata[1].strip("\n")
+    secrets_manager_client = get_secrets_manager_client(CLUSTER_REGION)
+    delete_s3_bucket(metadata_dict, secrets_manager_client)
+    delete_rds(metadata_dict, secrets_manager_client)
+
+
+def delete_s3_bucket(metadata_dict, secrets_manager_client):
+    s3_client = get_s3_client(CLUSTER_REGION)
+    s3_resource = boto3.resource("s3")
+    bucket_name = metadata_dict["bucket_name"]
+
+    print("Deleting S3 bucket...")
+
+    bucket = s3_resource.Bucket(bucket_name)
+    bucket.objects.all().delete()
+    s3_client.delete_bucket(Bucket=bucket_name)
+
+    buckets = s3_client.list_buckets()["Buckets"]
```

Comment:
lets remove this check because the previous command is sufficiant to say bucket is deleted